cocos/core/root.ts

```diff
@@ -484,7 +489,9 @@ export class Root {
         }
 
         const windows = this._windows;
-        const cameraList: Camera[] = [];
+        const cameraList = this._cameraList;
+        cameraList.splice(0, cameraList.length);
+
```

Comment:
cameraList is rebuilt every frame.

Reply:
Could we just use `cameraList.length = 0;` ?

Reply:
Yes. I forget we can do this.